internal/strict/strict.go

```diff
@@ -83,45 +85,49 @@ type Controls map[string]Control
 //nolint:lll,gochecknoglobals,stylecheck
 var StrictControls = Controls{
 	SpinUp: {
-		Error:   errors.Errorf("The `%s` command is no longer supported. Use `terragrunt run-all apply` instead.", SpinUp),
```

Comment:
Because this error is user facing, and it's more likely to involve multiple sentences don't you think it's better to have them be full sentences? If so, we should just add a `nolint` for whatever made you adjust this.

Reply:
Alternatively, maybe you know a special way to convert the error messages to proper casing before they are reported to users?

Reply:
Sure, I did what you suggested.

Reply:
> Alternatively, maybe you know a special way to convert the error messages to proper casing before they are reported to users?

Ah no, I don't know of any ready-made function that could do this, only manual message conversion, but I don't think it's worth it.